<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UCONVERTER</string>
  </property>
  <widget class="QPushButton" name="b_youtube">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Convertir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_url_youtube">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_youtube">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MP4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MP3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;UCONVERTER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>381</width>
     <height>15</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
